components/raft_log_engine/Cargo.toml

```diff
@@ -28,5 +28,6 @@ serde = "1.0"
 serde_derive = "1.0"
 kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
 raft = { version = "0.6.0-alpha", default-features = false }
-raft-engine = { git = "https://github.com/tikv/raft-engine" }
+# raft-engine = { git = "https://github.com/tikv/raft-engine" }
+raft-engine = { git = "https://github.com/hicqu/raft-engine", branch = "rewrite-cache-evictor" }
```

Comment:
Can this code be placed in the tikv org rather than a personal repo? I suggest allowing branch pushes to raft-engine since there are only a few contributors there. In my experience it is much better because it increases visibility (it just doesn't necessarily scale well for a repo like tikv/tikv).

Reply:
Sorry for disturbing you. When you saw the PR it's in WIP status, but I didn't add the label. Now it's ready for review and merge.